bazel/repository_locations.bzl

```diff
@@ -968,12 +968,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "compiler-rt",
         project_desc = "LLVM compiler runtime library",
         project_url = "https://compiler-rt.llvm.org",
-        version = "12.0.1",
-        sha256 = "b4c8d5f2a802332987c1c0a95b5afb35b1a66a96fe44add4e4ed4792c4cba0a4",
+        version = "14.0.0",
+        sha256 = "27ab7fcfb21d108093c0be766a9ed5fe18c04e4f74f936069711a312c8ae0377",
         # Only allow peeking at fuzzer related files for now.
         strip_prefix = "compiler-rt-{version}.src",
         urls = ["https://github.com/llvm/llvm-project/releases/download/llvmorg-{version}/compiler-rt-{version}.src.tar.xz"],
-        release_date = "2021-07-09",
+        release_date = "2022-03-23",
```

Comment:
I think you could just cherry-pick / use my commit, which has a better message: https://github.com/jwendell/envoy-1/commit/bf597f30f4e5d822305e5e5b03e50222195a8bbf

Reply:
And probably you should mention OSSM-3429 in the commit title (which my commit message does), which I think should not have been closed until the fix is actually merged.